gateway-provider-identity-assertion-common/src/test/java/org/apache/knox/gateway/identityasserter/filter/CommonIdentityAssertionFilterTest.java

```diff
@@ -74,6 +76,7 @@ public String mapUserPrincipal(String principalName) {
         return super.combineGroupMappings(mappedGroups, groups);
       }
     };
+    ThreadContext.put(MDC_AUDIT_CONTEXT_KEY, "dummy");
```

Comment:
I'm assuming this is for backward-compatibility. Is that correct?

Reply:
The test fails with an NPE if there is no AuditContext (auditService.getContext() returns null). It is just for making the test work.